packages/driver/src/cy/commands/cookies.ts

```diff
@@ -79,28 +79,16 @@ function cookieValidatesSecurePrefix (options) {
   return options.secure === false
 }
 
-interface InternalGetCookieOptions extends Partial<Cypress.Loggable & Cypress.Timeoutable> {
```

Comment:
I refactored out the need for these internal types by using locally scoped variables instead of mutating the `options` objects.